index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <a href="#" class="logo">Welcome</a>
+        <Nav id="menu">
+        <img src="./Images/icon-menu.png" alt="Menu Icon">
+        </Nav>
+    </header>
+<section class="bg">
+    <h1 class="hero up">Hey there. I’m <br>Usama</h1>
+    <h1 class="hero down">I'm a software<br>developer</h1>
+<p class="intro">
+    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+    don’t hestiate to contact me.
+</p>
+     <ul class="social-media">
+     <li><a href="#"><img src="./Images/social-media-icons/twitter.svg" alt="Link to Twitter"></a>
+    </li>
+     <li><a href="#"><img src="./Images/social-media-icons/Linkedin.svg" alt="Link to Linkedin"></a>
+    </li>
+     <li><a href="#"><img src="./Images/social-media-icons/medium.svg" alt="Link to Medium"></a>
+    </li>
+     <li><a href="#"><img src="./Images/social-media-icons/github.svg" alt="Link to Github"></a>
```

Comment:
- [Optional] Please consider using an indentation of two spaces all through your document. This is a convention and [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).